cleancat/base.py

```diff
@@ -311,7 +315,8 @@ def clean(self, value):
         return data
 
     def serialize(self, value):
-        return [self.field_instance.serialize(item) for item in value]
+        if value is not None:
+            return [self.field_instance.serialize(item) for item in value]
```

Comment:
I think we should return an empty list here by default. I'd rather not have two different cases (`[]` vs `None`) that we need to differentiate. If someone needs to distinguish between these they can override the method.

Reply:
I agree! Done in 9402238